web/ui/static/js/alerts.js

```diff
@@ -26,6 +26,47 @@ function init() {
         targetEl.removeClass('is-checked');
     }
   });
+
+  function displayAlerts(alertState, shouldDisplay) {
```

Comment:
Could this whole function be rewritten to

```js
  function displayAlerts(alertState, shouldDisplay) {
      $("#alertsTable > tbody > tr." + alertState).each(function(_, container) {
          $(container).toggle(shouldDisplay);
      });
  }
```

?

Reply:
sure, thanks!